routes/signup.js

```diff
@@ -0,0 +1,18 @@
+var httpRequest = require('request');
+
+module.exports = function(transaction, callback) {
```

Comment:
moved signup into a module because it's called in multiple spots now, because we do this inside stripe now too.